src/TCommandLine.cpp

```diff
@@ -569,63 +524,71 @@ void TCommandLine::slot_sendCommand(const char * pS)
 // You can cycle through all possible matches of the currently typed letters
 // with by repeatedly pressing tab or shift+tab. ESC-key brings you back into regular mode
 
-void TCommandLine::handleTabCompletion( bool direction )
+void TCommandLine::handleTabCompletion(bool direction)
 {
-    if( ( mTabCompletionCount < 0 ) || ( mUserKeptOnTyping ) )
-    {
+    if ((mTabCompletionCount < 0) || (mUserKeptOnTyping)) {
         mTabCompletionTyped = toPlainText();
-        if( mTabCompletionTyped.size() == 0 ) return;
+        if (mTabCompletionTyped.size() == 0) {
+            return;
+        }
         mUserKeptOnTyping = false;
         mTabCompletionCount = -1;
     }
     int amount = mpHost->mpConsole->buffer.size();
-    if( amount > 500 ) amount = 500;
+    if (amount > 500) {
+        amount = 500;
+    }
 
-    QStringList bufferList = mpHost->mpConsole->buffer.getEndLines( amount );
+    QStringList bufferList = mpHost->mpConsole->buffer.getEndLines(amount);
     QString buffer = bufferList.join(" ");
 
-    buffer.replace(QChar( 0x21af ), "\n");
-    buffer.replace(QChar('\n'), " " );
+    buffer.replace(QChar(0x21af), "\n");
```

Comment:
:earth_africa: That appear to be the '↯' codepoint {U+21af DOWNWARDS ZIGZAG ARROW in mathematical symbols, arrows}.  Is this going to be tolerable if that character CAN appear in the data stream in future versions of Mudlet.